### DISCUSSION TITLE
Tracking on moving sensor

### DISCUSSION THREAD

**Discussion by YiannisParask** [mikel-brostrom/yolo_tracking#942] [Q&A]
After testing some trackers (mostly ByteTrack and OCSort) with yolov5 as the detector, I figured that they perform well if the sensor is stationary. When applying the algorithm to a moving sensor they fail to re-id correctly.
Does anyone have experienced anything like this?
If yes, how did you face it?
Would something like stabilization work?

**Comment by mikel-brostrom** [comment]
Adding camera motion compensation could be very benefecial for your case
